stackdriver-debugger/src/com/google/cloud/tools/intellij/stackdriver/debugger/ProjectRepositoryValidator.java

```diff
@@ -129,7 +130,8 @@ public SyncResult checkSyncStashState() {
                     ServiceManager.getService(PluginInfoService.class)
                         .getClientVersionForCloudDebugger())
                 .execute();
-        for (Debuggee debuggee : debuggees.getDebuggees()) {
+        for (Debuggee debuggee :
```

Comment:
nit:, but would it be more readable here to just check if debuggees.getDebuggees() is null before entering the loop
?

Reply:
Probably, I agree.